docs/source/tutorials/create_and_submit_dataset.rst

```diff
@@ -48,7 +48,10 @@ instructions on how to assign values in this file.
       ],
     }
 
-2. Choose a data format as described in :ref:`dataset-formats`. This dataset uses the two-table
+2. Choose a data format as described in :ref:`dataset-formats`. Please note all requirements listed
```

Comment:
@ashreeta I added you as a reviewer to check the dataset submission additions. Please feel free to suggest additional content that would help dataset contributors. Also note the file dataset_formats.rst above.